src/presentation/pages/Login.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Navbar from '../components/Navbar'
+import { Button, Divider, Input } from '@nextui-org/react'
+import { Link } from 'react-router-dom'
+import LinkIcon from '../assets/LinkIcon'
+
+export default function Login() {
+  return (
+    <>
+    {/* FIX: acertar largura dos componentes na página e suas responsividades */}
+    {/* FEATURE: conectar ao banco e fazer checagens necessárias p/ funcionamento do botão de login*/}
+    {/* FIX: remover navbar? */}
+      <Navbar />
+        <main className="xs:mx-5 md:mx-20 lg:mx-72">
+          {/* TODO: adicionar logo */}
+          <header className="text-center text-4xl font-bold py-16">
+            Login
```

Comment:
Tudo que for texto deve estar dentro de uma tag que representa texto, como esse Login representa o titulo da sessão, sreia bom colocar dentro de um H1
```suggestion
            <h1 className="text-3xl font-bold">Login</h1>
```